backend/src/actions/startStreamAndCreateDashboard.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import startContainerMetricsStream from './docker/startContainerMetricsStream';
+import createGrafanaDashboardObject from './grafana/createGrafanaDashboardObject';
+import { GrafanaDatasource } from '../types';
+
+// Function that aggregates container info needed to get metrics and make dashboards.
+// Uses relevant container info and starts a metrics stream and creates a Grafana dashboard.
+export default async function startStreamAndCreateDashboard(
+  id: string,
+  containerName: string,
+  datasource: GrafanaDatasource
+) {
+  startContainerMetricsStream(id);
+  const dashboard = createGrafanaDashboardObject(id, containerName, datasource);
+
+  // Post request to Grafana API to create a dashboard using the returned dashboard object.
+  await axios.post(
```

Comment:
We should assign this to a variable and log an error to the console if the request is not successful (e.g. response status >=400)

Reply:
If the response indicates that the request was successful, **right here** is where we should log the "Grafana graphs for <container> are ready!" message.